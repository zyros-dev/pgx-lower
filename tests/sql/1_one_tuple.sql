LOAD 'pgx_lower.so';
SET client_min_messages TO NOTICE;
SELECT 'hello';
DROP TABLE IF EXISTS test;
CREATE TABLE test(id SERIAL);
INSERT INTO test(id) VALUES (42);
SELECT * FROM test;
