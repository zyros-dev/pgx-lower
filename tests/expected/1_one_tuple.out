do not define this file yet